src/store/metadata/stub/content/editions/a16mu_2023-11.json

```diff
@@ -0,0 +1,127 @@
+{
+  "@context": "https://staging.idpd.uk/#ns",
+  "@id": "https://staging.idpd.uk/datasets/a16mu/editions",
+  "@type": "hydra:Collection",
+  "title": "Modelled unemployment rate aged 16 years and over",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/cpih/editions/2022-01",
```

Comment:
so this needs to match the @id you've used to reference this edition within the "editions" field of the datasets. Think of it like the truncated (dataset level) and full (this level) view of the _same thing_. Where fields are shared they need to match.

Reply:
fixed